crypto/hash/htr/hashtree.go

```diff
@@ -1,17 +1,47 @@
 package htr
 
 import (
+	"runtime"
+	"sync"
+
 	"github.com/prysmaticlabs/gohashtree"
 )
 
+const minSliceSizeToParallelize = 5000
+
+func hashParallel(inputList [][32]byte, outputList [][32]byte, wg *sync.WaitGroup) {
```

Comment:
I wonder if you want to switch the arguments to `hashParallel(output, input)` to align with `gohashtree.Hash` to prevent silly mistake